nntrainer/models/model_common_properties.h

```diff
@@ -73,6 +73,17 @@ class SaveBestPath : public Property<std::string> {
   using prop_tag = str_prop_tag; /**< property type */
 };
 
+/**
+ * @brief train log save path property
+ *
+ */
+class SaveTrainLogPath : public Property<std::string> {
```

Comment:
For now it looks good, but I think this should be a callback so that the user has some control over the properties(https://www.tensorflow.org/guide/keras/custom_callback?hl=en)